smf/zone-network-setup/manifest.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+
+<service_bundle type='manifest' name='zone-network-setup'>
+
+<service name='oxide/zone-network-setup' type='service' version='1'>
+  <create_default_instance enabled='true' />
+
+  <!-- Run after the operating system's svc:/network/physical service is done. -->
+  <dependency name='physical' grouping='require_all' restart_on='none'
+    type='service'>
+  <service_fmri value='svc:/network/physical:default' />
+  </dependency>
+
+  <dependency name='multi_user' grouping='require_all' restart_on='none'
+    type='service'>
+  <service_fmri value='svc:/milestone/multi-user:default' />
+  </dependency>
+
+  <exec_method type='method' name='start'
+    exec='/opt/oxide/lib/svc/manifest/zone-network-setup.sh'
+    timeout_seconds='0' />
+  
+  <property_group name='startd' type='framework'>
+    <propval name='duration' type='astring' value='transient' />
+  </property_group>
+
+  <property_group name='config' type='application'>
+    <propval name='datalink' type='astring' value='unknown' />
+    <propval name='gateway' type='astring' value='unknown' />
+    <propval name='listen_addr' type='astring' value='unknown' />
```

Comment:
I think I'd be inclined to find a different name for this property as `listen_addr` is a property that the other services have and configures the address that they listen on. Perhaps just `static_addr` - it could even be a list of static addresses but I don't think there are any services that need that yet.

Reply:
Ah yeah, that property name makes much more sense. I'm not sure I'd like to have a list of addresses yet if none of the services require this. We could modify it later if the need arises. What do you think?